core/src/main/java/io/confluent/kafka/schemaregistry/masterelector/kafka/KafkaGroupMasterElector.java

```diff
@@ -115,7 +116,8 @@ public KafkaGroupMasterElector(SchemaRegistryConfig config,
       );
       List<String> bootstrapServers
           = config.getList(SchemaRegistryConfig.KAFKASTORE_BOOTSTRAP_SERVERS_CONFIG);
-      List<InetSocketAddress> addresses = ClientUtils.parseAndValidateAddresses(bootstrapServers);
+      List<InetSocketAddress> addresses = ClientUtils.parseAndValidateAddresses(bootstrapServers,
```

Comment:
I didn’t see this change as part of kip-302 (
https://github.com/apache/kafka/pull/4987/files) is there another pr where this parameter was added?

Reply:
KIP-235, I updated the PR title and description.